mypy/nodes.py

```diff
@@ -3283,6 +3423,8 @@ def f(x: B[T]) -> T: ...  # without T, Any would be used here
         "eager",
     )
 
+    __match_args__ = ("name", "target", "alias_tvars")
```

Comment:
You can add "no_args" probably for consistency with `TypeAliasExpr` (or remove it there).